README.md

```diff
@@ -181,7 +181,7 @@ Get security level that is supported on the current device with the current OS.
 
 > Note #1: `BIOMETRY_ANY`, `BIOMETRY_CURRENT_SET`, `BIOMETRY_ANY_OR_DEVICE_PASSCODE`, `BIOMETRY_CURRENT_SET_OR_DEVICE_PASSCODE` - recognized by Android as a requirement for Biometric enabled storage (Till we got a better implementation);
 >
-> Note #2: For Android we support only two states: `Default` (use the best available secured storage) and `Fingerprint` (use only biometric protected storage);
+> Note #2: For Android we support only two states: `None` (default) and `Fingerprint` (use only biometric protected storage);
```

Comment:
I'm not sure, but I think we should be having a third state that reflects on the previous `Default` behavior.

Reply:
Which I think would be analogous to `BIOMETRY_ANY` for iOS

Reply:
I wasn't thinking right here, this is correct 👍